UKHomeOffice/sgmr-service
# NMSW-34 Enable port persisting on refresh

## AC

Port field should persist it's last typed/selected value on page refresh for arrival and departure

----

## To test

- create a voyage
- on departure page type in and select a port
- refresh the page
- > port should persist
- click the link for the other port field
- type in the field
- refresh the page
- > port should persist what you typed in that field
- fill in fields and save and continue
- repeat the above for arrival

----

## Notes
